bcp/curate_matrices.py

```diff
@@ -7,7 +7,7 @@
 from cellxgene_schema.write_labels import AnnDataLabelAppender
 from urllib.parse import quote
 from cellxgene_ontology_guide.ontology_parser import OntologyParser
-ontology_parser = OntologyParser(schema_version="v6.0.0")
```

Comment:
I think just omit the schema_version parameter to get the current one, and avoid needing to update this (but confirm that works)